components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/layout/DefaultSuggestionViewHolder.kt

```diff
@@ -58,6 +64,12 @@ internal sealed class DefaultSuggestionViewHolder {
                 suggestion.onSuggestionClicked?.invoke()
                 selectionListener.invoke()
             }
+
+            val hasTitle = !title.isNullOrBlank()
```

Comment:
Do you mean leaving the visibility as `GONE` in XML? Who do you mean by consumer? The view holder that Fenix implements?

Reply:
Yeah, `GONE` in the XML, and we can set the visibility from Fenix's implementation. Let me get back to you on this, maybe there's a nicer way to do this.. 🤔

Reply:
OK, I've changed the XML in my commit right now.

Reply:
@cypressious I thought about this a bit more and let's leave the XML as always visible instead, sorry about the roundabout.

We should still remove this `hasTitle` check because it doesn't seem to add anything.